fmriprep/cli/run.py

```diff
@@ -275,8 +275,21 @@ def before_send(event, hints):
                     return None
                 elif re.match("Node .+ failed to run on host .+", msg):
                     return None
-            else:
-                return event
+
+            if 'breadcrumbs' in event and isinstance(event['breadcrumbs'], list):
```

Comment:
I think this does the same but seems more readable.
```Python
            if 'breadcrumbs' in event and isinstance(event['breadcrumbs'], list):
                fingerprints_to_propagate = ['no-disk-space', 'memory-error', 'permission-denied',
                                             'keyboard-interrupt']
                for bc in event['breadcrumbs']:
                    msg = bc.get('message', 'empty-msg')
                    if msg in fingerprints_to_propagate:
                        event['fingerprint'] = [msg]
```

Reply:
nice!